source/nims/index.md

```diff
@@ -0,0 +1,22 @@
+---
+review_priority: "index"
+html_theme.sidebar_secondary.remove: true
+---
+
+# NIMs Microservices
```

Comment:
I think we should try and match the exact wording that is being used on nvidia.com. They seem to refer to them as `NVIDIA NIM Microservices`.

https://docs.nvidia.com/nim/index.html

```suggestion
# NVIDIA NIM Microservices
```